module/src/components/calendarDisplay/calendarDisplay.component.tsx

```diff
@@ -0,0 +1,190 @@
+import * as React from 'react';
+
+import { Calendar, Select } from '../..';
+import { IBindingProps } from '../../hooks/form';
+import { Arrays } from '../../utils/arrays';
+import { ClassNames } from '../../utils/classNames';
+import { Dates } from '../../utils/dates';
+import { Maths } from '../../utils/maths';
+import { Button } from '../button';
+import { getDayOfWeekHeadings, getDaysWithDisplayFormat } from './calendarDisplay.utils';
```

Comment:
All of my utils files so far have returned a namespace, we should probably figure something out for this to be consistent

Reply:
So have mine when they're general utils inside the `utils` folder, but surely that's not necessary when they're utils specific to a single component?